.github/workflows/ci-build.yml

```diff
@@ -58,7 +53,10 @@ jobs:
       run: npm ci
       working-directory: ./vscode-extension
     - name: Install local lsp
-      run: npm i ../output/fabric8-analytics-lsp-server-*.tgz
+      run: npm i ../fabric8-analytics-lsp-server-*.tgz
+      working-directory: ./vscode-extension
+    - name: Copy fabric8-analytics-lsp-server files to one level up(FIXME)
```

Comment:
This is a temporary workaround, will be removed once vscode extension starts using lsp from new path. i.e. `node_modules/fabric8-analytics-lsp-server/dist/`